src/components/Table/Cells.tsx

```diff
@@ -0,0 +1,45 @@
+import { PropsWithChildren } from 'react'
+import styled from 'styled-components'
+import { ClickableStyle, LinkStyle, ThemedText } from 'theme'
+
+const StyledCell = styled.div<{ justifyContent?: string; color?: string }>`
+  justify-content: ${({ justifyContent }) => justifyContent ?? 'right'};
+  color: ${({ color }) => color};
+  padding: 14px 6px 14px 6px;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 64px;
+`
+const Link = styled.a`
```

Comment:
externalLnik here?

Reply:
Using the shared `ExternalLink` component now, thank you for suggesting